featurebyte/feast/utils/registry_construction.py

```diff
@@ -21,7 +21,6 @@
 from feast.protos.feast.core.Registry_pb2 import Registry as RegistryProto
 from feast.repo_config import RegistryConfig, RepoConfig
 
-from featurebyte.common.env_util import add_sys_path
```

Comment:
Is the `add_sys_path` helper function still needed elsewhere?

Reply:
Yes, but it is used in the test codes only. Will move it to test module in the coming PR: https://github.com/featurebyte/featurebyte/blob/aed8365128b4359ab5aeeef05fd24a8a60a6b592/tests/util/helper.py#L586-L598